libs/platform/src/lib/table/table.component.scss

```diff
@@ -55,17 +55,6 @@ $fd-table-header-color-var: var(--sapList_HeaderBackground, #f2f2f2);
         border-spacing: 0;
     }
 
-    /** Temporary until arrow keyboard navigation not implemented */
```

Comment:
is there a ticket for that?

Reply:
These body focus styles are placed here for the page-scrolling feature. So we can focus on the table body and see the outline and also can use arrows to scroll rows. In #5692 we're going to change this behavior but I'm not sure we should change it here.

Reply:
these styles were removed as per https://github.com/SAP/fundamental-ngx/issues/7011

Reply:
Before: 

<img width="982" alt="Screenshot 2021-11-02 at 14 42 53" src="https://user-images.githubusercontent.com/20265336/139848677-56829434-883d-4f09-8ef7-3a9919e8e162.png">

After:

<img width="981" alt="Screenshot 2021-11-02 at 14 42 25" src="https://user-images.githubusercontent.com/20265336/139848689-7eb34601-a77b-47b7-8f69-057c81e59e92.png">

Reply:
got it. my mistake

Reply:
@platon-rov @droshev brought these styles back